utils.py

```diff
@@ -386,10 +422,10 @@ def require_valid_name(name, name_type, allow_empty=False):
     """Generic name validation.
 
     Args:
-      name: the name to validate.
-      name_type: a human-readable string, like 'the exploration title' or
-        'a state name'. This will be shown in error messages.
-      allow_empty: if True, empty strings are allowed.
+        name: str. the name to validate.
+        name_type: str. a human-readable string, like 'the exploration title' or
+            'a state name'. This will be shown in error messages.
+        allow_empty: bool. if True, empty strings are allowed.
```

Comment:
Capitalize the first letter in all three cases. e.g.- the --> The

Reply:
Please take care that all the comments are addressed in future. Here, comment is not addressed.